lib/tasks/data_hygiene.rake

```diff
@@ -40,4 +40,31 @@ namespace :data_hygiene do
   task :bulk_update_organisation, %i[csv_filename] => :environment do |_, args|
     DataHygiene::BulkOrganisationUpdater.call(args[:csv_filename])
   end
+
+  desc "Discard drafts (if present) and unpublish"
+  task :discard_drafts_and_unpublish, %i[content_id] => :environment do |_, args|
+    abort("Missing parameter: content_id") if args[:content_id].blank?
+
+    documents = Document.where(content_id: args[:content_id])
+    abort("Content ID #{args[:content_id]} not found") unless documents.count.positive?
```

Comment:
Would `any?` here be a shorter way of achieving the same?

```suggestion
    abort("Content ID #{args[:content_id]} not found") unless documents.any?
```

Reply:
Ah yes, good point. I'll accept and then squish the commits.